js/core/component.js

```diff
@@ -125,9 +125,15 @@ const Component = Class.inherit({
         });
     },
 
-    _logDeprecatedWarning(option, info) {
-        const message = info.message || (`Use the '${info.alias}' option instead`);
-        errors.log('W0001', this.NAME, option, info.since, message);
+    _logDeprecatedWarning(error, options) {
```

Comment:
Why we need common method to warn about deprecated widgets and deprecated options? As I can see this method does not contain any common logic. And it uses string comparison to find out what to do. Not cool.

Reply:
1. The `_logDeprecatedWarning` doesn't contain the option word so it has a common name for warnings.
I can agree to create a new method for a deprecated component and rename the `_logDeprecatedWarning` to `_logDeprecatedOptionWarning` as an example.
2. Comparison of a string?  The error argument is an id of warning

Reply:
1. Let's do it
2. I know. But we do not need redundant string comparison in case we have two methods.